From 479c4e26cedae2d937a7933e35798d3419a893bd Mon Sep 17 00:00:00 2001
From: "Jan Alexander Steffens (heftig)" <jan.steffens@gmail.com>
Date: Mon, 23 Sep 2019 18:14:58 +0000
Subject: [PATCH] nautilus-previewer: Allow auto-starting previewer on call

Using DO_NOT_AUTO_START_ON_CONSTRUCTION gets the desired behavior
of not auto-starting the previewer at proxy construction, but allow
auto-starting at the first ShowFile call.

Fixes https://gitlab.gnome.org/GNOME/nautilus/issues/1229
---
 src/nautilus-previewer.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/nautilus-previewer.c b/src/nautilus-previewer.c
index 0511dc991..40a02522e 100644
--- a/src/nautilus-previewer.c
+++ b/src/nautilus-previewer.c
@@ -48,7 +48,7 @@ ensure_previewer_v2_proxy (void)
         GDBusConnection *connection = g_application_get_dbus_connection (g_application_get_default ());
 
         previewer_v2_proxy = g_dbus_proxy_new_sync (connection,
-                                                    G_DBUS_PROXY_FLAGS_DO_NOT_AUTO_START,
+                                                    G_DBUS_PROXY_FLAGS_DO_NOT_AUTO_START_AT_CONSTRUCTION,
                                                     NULL,
                                                     PREVIEWER_DBUS_NAME,
                                                     PREVIEWER_DBUS_PATH,
-- 
2.22.0

